tensorflow/core/grappler/costs/graph_properties.cc

```diff
@@ -783,13 +783,13 @@ class SymbolicShapeRefiner {
       int src_output = fanin.port_id;
       const NodeDef* src = fanin.node;
       NodeContext* src_ctx = GetNodeContext(src);
-      InferenceContext* src_ic = src_ctx->inference_context.get();
       if (src_ctx == nullptr) {
         return errors::FailedPrecondition(
-            "Input ", dst_input, " ('", src->name(), "') for '", node->name(),
```

Comment:
Hi @rmlarsen, do you know how `src` can be null?

Reply:
It seems to affect certain models using TF-TRT in TF1.14 only (TF1.13 worked)

Reply:
Is `src` null or `src_ctx`? Or both?

Reply:
Both are null, src was null first which cause src_ctx to be null when it is created